### ISSUE TITLE
Uncaught TypeError: Cannot read property 'data' of undefined

### ISSUE THREAD

**Issue by BorislavZlatanov** [CalderaWP/Caldera-Forms#1326]
## What Version Of Caldera Forms, WordPress and PHP Are You Using?

WordPress Version: 4.7.2PHP Version: 5.6.30MySQL Version: 5.6.30Caldera Forms Version: 1.5.0.3WP_DEBUG: 1
## Does Your Issue Presist When You Disable All Other Plugins and Switch To The Default Theme?
There seems to be a conflict with the Pods plugin (I have version 2.6.8). When I deactivate Pods, the javascript error disappears. I also have Caldera Easy Pods installed (version 1.1.10).
## What Is The Unexpected Beahviour?

I filled out the form and clicked Submit. Nothing happened.
## What PHP Errors Have You Logged While Reproducing This Bug?

No PHP errors or notices related to this.
## What JavaScript Errors Have You Seen While Reproducing This Bug?

The error below is from the Chrome console. The error comes up when entering a text in one of the form fields. I don't get any 404s from the Network tab so nothing indicates that a javascript file might not have been loaded.

ajax-core.min.js?ver=1.5.0.3:1 Uncaught TypeError: Cannot read property 'data' of undefined
    at Object.error (ajax-core.min.js?ver=1.5.0.3:1)
    at Object.request_error (jquery-baldrick.min.js?ver=1.5.0.3:1)
    at c (jquery-baldrick.min.js?ver=1.5.0.3:1)
    at Object.error (jquery-baldrick.min.js?ver=1.5.0.3:1)
    at i (jquery.js?ver=1.12.4:2)
    at Object.fireWith [as rejectWith] (jquery.js?ver=1.12.4:2)
    at y (jquery.js?ver=1.12.4:4)
    at Object.abort (jquery.js?ver=1.12.4:4)
    at Object.request (jquery-baldrick.min.js?ver=1.5.0.3:1)
    at c (jquery-baldrick.min.js?ver=1.5.0.3:1)
error	@	ajax-core.min.js?ver=1.5.0.3:1
request_error	@	jquery-baldrick.min.js?ver=1.5.0.3:1
c	@	jquery-baldrick.min.js?ver=1.5.0.3:1
error	@	jquery-baldrick.min.js?ver=1.5.0.3:1
i	@	jquery.js?ver=1.12.4:2
fireWith	@	jquery.js?ver=1.12.4:2
y	@	jquery.js?ver=1.12.4:4
abort	@	jquery.js?ver=1.12.4:4
request	@	jquery-baldrick.min.js?ver=1.5.0.3:1
c	@	jquery-baldrick.min.js?ver=1.5.0.3:1
(anonymous)	@	jquery-baldrick.min.js?ver=1.5.0.3:1
dispatch	@	jquery.js?ver=1.12.4:3
r.handle	@	jquery.js?ver=1.12.4:3
trigger	@	jquery.js?ver=1.12.4:3
a.event.trigger	@	jquery-migrate.min.js?ver=1.4.1:2
(anonymous)	@	jquery.js?ver=1.12.4:3
each	@	jquery.js?ver=1.12.4:2
each	@	jquery.js?ver=1.12.4:2
trigger	@	jquery.js?ver=1.12.4:3
(anonymous)	@	(index):1085

Any suggestions on how to resolve this conflict with the Pods plugin?  
Thanks!

**Comment by Shelob9** [comment]
@BorislavZlatanov  Thanks for using Caldera Forms. Sorry about this issue.

That JavaScript error means "something" happened on the server and made an error. To know what that something is, you will need to turn on debug logging. See: https://calderaforms.com/2016/05/wordpress-debug-logging/

If you can re-run the submission with debug logging on, and tell me what errors you get that would be very helpful.

**Comment by BorislavZlatanov** [comment]
@Shelob9 Thanks for your help, it pointed me in the right direction.

I had debug logging turned on, but I didn't have any related PHP errors. But the information you gave me about the problem happening at the server level pointed me to noticing the AJAX call that was being made after I put in something in a form's field. I don't know how I didn't notice those AJAX calls before. Anyways, it was returning no responses, or unexpected (for me) responses with data from a Pod that I had set up. This made me remember that some time ago I had connected the Caldera Form with an Easy Pod, but I wasn't actually using that connection. Up till now, it wasn't causing any problem, but it seems that after the Caldera Forms 1.5 update it started giving me the above problem. So I disconnected the unused Easy Pod from the form and no AJAX calls were being made anymore upon inputting something in a field. And consequently, the javascript error was gone. The form works fine now.

Thanks again.
